CODEOWNERS

```diff
@@ -25,5 +25,5 @@ test/OpenTelemetry.Contrib.Instrumentation.MassTransit.Tests/               @ope
 test/OpenTelemetry.Contrib.Instrumentation.Owin.Tests/                      @open-telemetry/dotnet-contrib-approvers @codeblanch
 test/OpenTelemetry.Contrib.Instrumentation.Wcf.Tests/                       @open-telemetry/dotnet-contrib-approvers @codeblanch
 test/OpenTelemetry.Contrib.Instrumentation.MySqlData.Tests/                 @open-telemetry/dotnet-contrib-approvers @moonheart
-src/OpenTelemetry.Contrib.PersistentStorage.Tests/                          @open-telemetry/dotnet-contrib-approvers @vishweshbankwar
+src/OpenTelemetry.Contrib.Extensions.PersistentStorage.Tests/                          @open-telemetry/dotnet-contrib-approvers @vishweshbankwar
```

Comment:
Similar to https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/181/files#r792171120